src/alto-ui/Alert/Alert.scss

```diff
@@ -51,18 +51,14 @@ $alert-text-margin-top: $alert-height / 2 - $alert-text-line-height / 2;
   }
 }
 
-.Alert__icon,
-.Alert__close {
-  transform: translateY(calc(#{$alert-height / 2} - 50%));
-}
```

Comment:
Did you try on long text (there is a story for this). icon and close icon should stay at the top

Reply:
Fixed thx